AllReadyApp/Web-App/AllReady/Services/MessageServices.cs

```diff
@@ -22,6 +22,7 @@ public Task SendEmailAsync(string email, string subject, string message)
                 {
                     EmailMessage = message,
                     EmailRecipients = new List<string> {email},
+                    HtmlMessage = message,
```

Comment:
@tonysurma This is the one that addresses the HTML aspect, the link not rendering in the email.